cpg_workflows/stages/genotyping/genotype.py

```diff
@@ -10,15 +10,15 @@
     SequencingGroupStage,
     StageInput,
     StageOutput,
+    Workflow,
+    WorkflowError,
     stage,
 )
 
-from .. import get_batch
-from .align import Align
+from ... import get_batch
```

Comment:
not a fan of this at all. We're installing cpg_workflows as a package in the image we intend to run. Relative imports are garbage.

This is also the wrong import, we're using `cpg_utils.hail_batch.get_batch`
```suggestion
from cpg_utils.hail_batch import get_batch
```